[Analytics] Small updates to WAE get started page (#8712)
cloudflare/cloudflare-docs: content/analytics/analytics-engine/get-started.md

]
```

> Save the changes that you made to your `wrangler.toml` file. Republish your worker by running `wrangler publish` from the Terminal window to update the changes. In the dashboard, you can also verify if your deployment was successful.

Reviewer: ```suggestion
Save the changes that you made to your `wrangler.toml` file. Republish your Worker by running `wrangler publish` from the Terminal window to update the changes. In the dashboard, you can also verify if your deployment was successful.
```

  }
```

Besides writing static data points, a common use case of Workers Analytics Engine is to capture information about incoming HTTP requests.

In the runtime API documentation, you can find  information about the [variables](/workers/runtime-apis/request/) that are available in the runtime. Because these variables are already available in the runtime, you will not need to define them. In other words, you can directly use `request.cf.<variable name>` as a blob or double field. Using this adaptation of this Worker [example](/workers/examples/geolocation-hello-world/) template, you can write the geolocation variables to Analytics Engine.

```js
env.EXAMPLE_DATASET.writeDataPoint({

>     'blobs': [ 
>         request.cf.colo, 
>         request.cf.country, 
>         request.cf.city, 
>         request.cf.region, 
>         request.cf.timezone],
>     'doubles': [
>         request.cf.metroCode, 
>         request.cf.longitude, 
>         request.cf.latitude, 
>     ],
>     'indexes': [
>         request.cf.postalCode
>     ] 
> });

Reviewer 2: Sorry, I just realised my original suggestion isn't quite perfect :-)

```suggestion
'blobs': [ 
    request.cf.colo, 
    request.cf.country, 
    request.cf.city, 
    request.cf.region, 
    request.cf.timezone
  ],
  'doubles': [
    request.cf.metroCode, 
    request.cf.longitude, 
    request.cf.latitude
  ],
  'indexes': [
    request.cf.postalCode
  ] 
});
```

  }
```

Besides writing static data points, a common use case of Workers Analytics Engine is to capture information about incoming HTTP requests.

> In the runtime API documentation, you can find  information about the [variables](/workers/runtime-apis/request/) that are available in the runtime. Because these variables are already available in the runtime, you will not need to define them. In other words, you can directly use `request.cf.<variable name>` as a blob or double field. Using this adaptation of this Worker [example](/workers/examples/geolocation-hello-world/) template, you can write the geolocation variables to Analytics Engine.

Reviewer: ```suggestion
In the runtime API documentation, you can find  information about the [variables](/workers/runtime-apis/request/). Because these variables are already available in the runtime, you will not need to define them. In other words, you can directly use `request.cf.<variable name>` as a blob or double field. Using this adaptation of this Worker [example](/workers/examples/geolocation-hello-world/) template, you can write the geolocation variables to Analytics Engine.
```

```js

> env.EXAMPLE_DATASET.writeDataPoint({

Reviewer: ```suggestion
env.<EXAMPLE_DATASET>.writeDataPoint({
```

The SQL API is available as an HTTP endpoint at `https://api.cloudflare.com/client/v4/accounts/YOUR_ACCOUNT_ID/analytics_engine/sql` using the `POST` and `GET` method. You need to include an `Authorization: Bearer _____` token where the underscores should be replaced with a Cloudflare [API Token](https://dash.cloudflare.com/profile/api-tokens) that has the `Account Analytics Read` permission.

> If you prefer a web interface, you can use [Postman](https://www.postman.com/) to connect to the SQL API endpoint and run your query. Postman is an API platform that can be used to test SQL APIs. You can use the endpoint mentioned above using the `POST` and `GET` method.

Reviewer 2: The description of Postman feels slightly "off" to me. How about:

```suggestion
If you prefer a graphical interface, you can use [Postman](https://www.postman.com/) to connect to the SQL API endpoint and run your query. Postman is an application that can be used to test APIs. You can use the endpoint mentioned above using the `POST` and `GET` methods.
```
